### DISCUSSION TITLE
CMA-persmission-denied

### DISCUSSION THREAD

**Discussion by sinatahmooresi** [Nek5000/nekRS#478] [General]
Hello, 
Parallel running using $nekbmpi <case> <rank> does not work on my bash ubuntu. I will get an error the central part of which is: MCA-permission-denied. Are there similar experiences on this? I found a couple of threads on the same problem with parallel jobs in OpenFOAM while they changed their WSL from version 1 to 2. 

Thanks

**Comment by stgeke** [comment]
I think this is related to your MPI not nekRS.
